hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapDestroyCacheMessageTask.java

```diff
@@ -86,7 +86,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "destroyCache";
```

Comment:
There is no QueryCache.destroy() that the user can use to call this message task. Or they can call IMap.destroy as well. I think destroyQueryCache is a better name.

Reply:
It is called through [this](https://github.com/hazelcast/hazelcast/blob/v5.3.0/hazelcast/src/main/java/com/hazelcast/map/QueryCache.java#L226) API. It is NOT IMap method.

Reply:
right, but it can also be called through IMap.destroy. Lets follow task name.